src/mongocxx/test/pool.cpp

```diff
@@ -96,6 +97,7 @@ TEST_CASE(
     REQUIRE(interposed.crl_file == crl_file);
     REQUIRE(interposed.weak_cert_validation == allow_invalid_certificates);
 }
+#endif
```

Comment:
I filed https://jira.mongodb.org/browse/CXX-778 to track the above idea. I think for now we should move forward with this as is.

@samantharitter - Should I leave the error TODOs, or hold this until after you and @tessavitabile  have the updated error handling in?